packages/material-ui-system/src/index.spec.tsx

```diff
@@ -48,20 +48,20 @@ function cssTest() {
  * This is not equivalent to the implementation. Ideally `css` would be optional
  * but that breaks system/basics/#css-property
  */
```

Comment:
Reference to `css` and system/basics/#css-property

Reply:
I've replaced this and replaced the `css-property` example with `sx-property` one

Reply:
I'm not seeing any changes in the code comment above.

https://github.com/mui-org/material-ui/pull/23480/files/ab992fbee15c0b203d0192dd65d86e04b28a7900#diff-378c9e993f4aabdb6ed835cdfbcb8474ccf7c1961152f22647350a64627dc2d9L48-L50